### ISSUE TITLE
This method is even worse than direct resizing.

### ISSUE THREAD

**Issue by TianDekuan** [lzhxmu/AccDiffusion_v2#3]
Hi, thanks for sharing the code!

I tested many cases using the released implementation. The method can indeed generate 4K results, but in my experiments, the visual quality appears lower than:

generating 1K images with SD-XL and then upscaling them to 4K.

In my tests, the approach (1K → resize to 4K) often produced noticeably sharper and more consistent results than the method in the paper.

I may be misunderstanding some settings or missing recommended hyperparameters. Could you please clarify:

Are there specific configurations or tricks needed to reproduce the paper's best performance?

Does the method have known limitations in certain scenes (e.g., faces, textures)?

Is there an official pretrained checkpoint or demo script that best reflects the paper's results?

I would really appreciate any guidance, as I would like to fully understand and reproduce the expected performance. Thanks again for the work and for making the code public!

Here are some of my test examples:
![Image](https://github.com/user-attachments/assets/fe388e23-ba9e-4019-85bb-b9d49b63f096)
1K

![Image](https://github.com/user-attachments/assets/c6f0f96f-1f14-4f15-99fb-826d45421533)
accv2 4K

**Comment by lzhxmu** [comment]
Thank you very much for your interest in our work and for taking the time to test the released implementation — we really appreciate your detailed feedback and comparison experiments.

To clarify, AccDiffusion v2 is built upon the DemoFusion framework for high-resolution image generation. Its main goal is to alleviate local distortions and reduce repeated generation of small objects that may occur in DemoFusion-based pipelines.

In practice, we recommend the following usage strategy:

You may first try using DemoFusion directly.

If you observe local artifacts or repeated small objects, you can then enable the parameters proposed in AccDiffusion v2 — namely c, use_md_prompt, and use_controlnet — to mitigate these issues.

Otherwise, if such distortions are not present, using the default DemoFusion setup will often yield the best fine-detail sharpness.

Please note that the optimal hyperparameters (especially c and controlnet_conditioning_scale) may vary across prompts and even different seeds, so it is often necessary to perform several trials to find the best combination for a given input.

Finally, since image generation techniques are progressing rapidly, we also recommend exploring recent models with DiT-based architectures that natively support 4K generation — these can often achieve even better high-resolution results.

Thank you again for your thoughtful feedback and for your support of our work!

**Comment by TianDekuan** [comment]
Thank you for your reply!

I would like to clarify my question — it is not about comparison with DemoFusion. Instead, my concern is why the proposed method performs worse than simple direct resizing.

To avoid repetitive patterns, I first generate images at a lower resolution, then resize them to 4K. This strategy indeed reduces repetition. However, direct resizing usually causes noticeable blur and loss of detail. The primary motivation of this task, is to recover high-resolution details beyond what direct resizing can offer through diffusion-based refinement.

In my experiments, though, the proposed method introduces severe boundary blurring on 4K outputs and often looks worse than resizing a 1K SD-XL image to 4K. This raises a concern:

If the results are inferior to direct resizing, how do we justify the practical value of the method?

Thanks again for your time!

**Comment by lzhxmu** [comment]
We recommend reducing the value of c (c=0.1 or c=0) and turning off --use_controlnet when the output does not exhibit noticeable repetition or local distortion, as this typically leads to sharper boundaries and improved detail preservation.

The main contribution of our method is to address the issues of repeated objects and local distortions that sometimes occur in DemoFusion during high-resolution generation. In most cases, DemoFusion produces finer details than direct resizing, especially for structures like animal fur and other intricate textures.

**Comment by lzhxmu** [comment]
<img width="855" height="297" alt="Image" src="https://github.com/user-attachments/assets/f9816b5a-90d7-4466-a57d-a4f73a85188a" />
As depicted in the Figure of our paper, AccDiffusion v2 excels in generating intricate details like the cat’s eye but may introduce irrelevant elements such as superfluous “PPP” details.  This is a common issue encountered in the training-free patch-wise diffusion extrapolation methods, and it requires joint efforts from the research community to further enhance these methods.

**Comment by TianDekuan** [comment]
Thank you for your reply.  I also observed noticeable jagged edges and artifacts (e.g., “PPP”-like patterns). These artifacts significantly affect the visual quality and, in many cases, lead to results that are inferior to directly re
sizing a 1K image to 4K. Additionally, this phenomenon seems to occur frequently. In my current tests, I have not yet found examples where detail improvement is achieved without noticeable jagged-edge artifacts, which tend to affect the overall visual quality. Below are some of my test results for reference.
![Image](https://github.com/user-attachments/assets/9321522f-87dd-458a-90d5-7a5055a4104b)
1K

![Image](https://github.com/user-attachments/assets/cc59979c-c211-4a7b-bdca-77e5d1859cd8)
accv2-4K

![Image](https://github.com/user-attachments/assets/aa9ea81f-e1f8-4311-bbcb-2738c3a7eaf4)
1k

![Image](https://github.com/user-attachments/assets/a6b57847-ae75-4776-a05e-3bee8964e639)
accv2-4k


I will try again after adjusting the hyperparameters as you suggested. Thank you for your support.

**Comment by lzhxmu** [comment]
If you have any questions, please contact me via email (lzhedu@foxmail.com) for more efficient communication.
